kilted/distribution.yaml

```diff
@@ -1314,6 +1314,21 @@ repositories:
       url: https://github.com/luxonis/depthai-core-release.git
       version: 3.0.0-4
     status: developed
+  depthai-ros:
+    release:
+      packages:
+      - depthai-ros
+      - depthai_bridge
+      - depthai_descriptions
+      - depthai_examples
+      - depthai_filters
+      - depthai_ros_driver
+      - depthai_ros_msgs
+      tags:
+        release: release/kilted/{package}/{version}
+      url: https://github.com/luxonis/depthai-ros-release.git
+      version: 3.0.0-1
+    status: developed
```

Comment:
Would you be willing to add a `source` entry and a `doc` entry?

Reply:
You can just copy those from the Jazzy distribution file and just update the `version` to `kilted`: https://github.com/ros/rosdistro/blob/2fd6386401f2e0495e4c0c4e58c447b6b74b92d4/jazzy/distribution.yaml#L1828-L1851

Reply:
Fixed in https://github.com/ros/rosdistro/pull/47208